### ISSUE TITLE
Domain expired

### ISSUE THREAD

**Issue by mgorny** [ekg2/ekg2#56]
It seems that ekg2.org expired.

**Comment by leafnode** [comment]
...and someone has already registered this domain. So, do we need another one?

**Comment by mgorny** [comment]
I don't think so. We can use the GitHub project for homepage and/or put homepage on ekg2.github.com or sth like that.

**Comment by leafnode** [comment]
+1 for that. I'll try to convert the sources of the website to something usable.

**Comment by floren** [comment]
Highly recommend you modify the URL in the repo, because ekg2.org is now a pornography site.

**Comment by pjona** [comment]
Maybe someone with admin permission can enable GitHub Page for this repository and we will have a new URL?

**Comment by mgorny** [comment]
@leafnode do you have admin powers on the project? We need to remove the old homepage from the project page, given the new content (NSFW).

**Comment by leafnode** [comment]
Done. Will close the issue after moving the page to GitHub.
